holidays/locale/uk/LC_MESSAGES/FI.po

```diff
@@ -86,3 +86,69 @@ msgstr "Різдво Христове"
 #. Second Day of Christmas.
 msgid "Tapaninpäivä"
 msgstr "Другий день Різдва"
+
+#. Day of Finnish architecture and design, Birthday of Alvar Aalto.
+msgid "Arkkitehtuurin ja muotoilun päiviä"
+msgstr "Дні архітектури та дизайну"
+
+#. Birthday of the national poet Johan Ludvig Runeberg
+msgid "Runebergin päivä"
+msgstr "День народження народного поета Йохана Людвіга Рунеберга"
+
+#. National Veterans' Day
+msgid "Kansallinen veteraanipäivä"
+msgstr "Національний день ветеранів"
+
+#. Europe Day
+msgid "Eurooppa-päivä"
+msgstr "День Європи"
+
+#. Day of Finnish Identity (or The Finnish Identity Day)
+msgid "Suomalaisuuden päivä"
+msgstr "День народження державного діяча Йохана Вільгельма Шелмана"
+
+#. Remembrance Day
+msgid "Kaatuneitten muistopäivä"
+msgstr "День ветеранів Національної війни"
+
+#. Birthday of the poet Eino Leino
+msgid "Eino Leinon päivä"
+msgstr "День народження поета Ейно Лейно"
+
+#. Finland’s Nature Day
+msgid "Suomen luonnon päivä"
+msgstr "День природи Фінляндії"
+
+#. Miina Sillanpää Day, Day of Civic Participation
+msgid "Miina Sillanpään ja kansalaisvaikuttamisen päivä"
+msgstr "День Міїни Сілланпяя та громадянської активності."
+
+#. Birthday of the National writer Aleksis Kivi
+msgid "Aleksis Kiven päivä"
+msgstr ""
+"День народження народного письменника Алексіса Ківі, День фінської "
+"літератури"
+
+#. United Nations Day
+msgid "Yhdistyneiden Kansakuntien päivä"
+msgstr "День Організації Об'єднаних Націй"
```

Comment:
```suggestion
msgstr "День Організації Обʼєднаних Націй"
```

Reply:
Question -- why would we want to use `ʼ` here in `Обʼєднаних Націй`, but use `'` in `Finland's Nature Day` (as per your suggestion above)?

Reply:
It's because of different languages/layouts. Here is some AI generated explanation:

```
The Ukrainian language uses a specific apostrophe character (ʼ) distinct from the typical straight apostrophe ('). The Ukrainian apostrophe is a non-vowel sound and is used in words where it signifies a slight separation between consonant sounds, often following consonants and preceding the soft sign (ь) or certain vowels.

The specific character for the Ukrainian apostrophe is ʼ (U+02BC), which is different from the ASCII apostrophe (', U+0027).
```

Reply:
Oooh, I see, TIL ✅